block/manager.go

```diff
@@ -934,19 +936,25 @@ func (m *Manager) submitBlocksToDA(ctx context.Context) error {
 				gasPrice = gasPrice * m.dalc.GasMultiplier
 			}
 			m.logger.Info("retrying DA layer submission with", "backoff", backoff, "gasPrice", gasPrice, "maxBlobSize", maxBlobSize)
-			time.Sleep(backoff)
-			backoff = m.exponentialBackoff(backoff)
+
 		case da.StatusTooBig:
-			m.logger.Error("DA layer submission failed", "error", res.Message, "attempt", attempt)
 			maxBlobSize = maxBlobSize / 4
-			time.Sleep(backoff)
-			backoff = m.exponentialBackoff(backoff)
+			fallthrough
 		default:
 			m.logger.Error("DA layer submission failed", "error", res.Message, "attempt", attempt)
-			time.Sleep(backoff)
+
+		}
+
+		if res.Code != da.StatusSuccess {
 			backoff = m.exponentialBackoff(backoff)
 		}
```

Comment:
The `exponentialBackoff` code needs to be under `default` case otherwise we will overwrite the custom backoff from `case da.StatusNotIncludedInBlock, da.StatusAlreadyInMempool`

Reply:
That makes sense. Thanks for pointing it out @MSevey, updated the pr